lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -115,12 +115,32 @@
                     return DateUtils.localize(context);
                 },
 
+                getCertificatePriceString: function(run) {
+                    var verifiedSeat, currency;
+                    if ('seats' in run && run.seats.length) {
+                        // eslint-disable-next-line consistent-return
+                        verifiedSeat = _.filter(run.seats, function(seat) {
+                            if (['verified', 'professional', 'credit'].indexOf(seat.type) >= 0) {
+                                return seat;
+                            }
+                        })[0];
+                        currency = verifiedSeat.currency;
+                        if (currency === 'USD') {
+                            return '$' + verifiedSeat.price;
+                        } else {
+                            return verifiedSeat.price + ' ' + currency;
+                        }
+                    }
+                    return null;
+                },
+
                 setActiveCourseRun: function(courseRun, userPreferences) {
                     var startDateString,
                         courseImageUrl;
 
                     if (courseRun) {
-                        if (courseRun.advertised_start !== undefined && courseRun.advertised_start !== 'None') {
+
+                        if (!([undefined, 'None', null].indexOf(courseRun.advertised_start) >= 0)) {
```

Comment:
Yeah, you're right. The 'None' value is a weird one. I think it would be clearer what's going on in the code if you created a function called something like ```isDefined``` that does the check i.e.
```
isDefined: function(str) {
    return [undefined, 'None', null].indexOf(str) >= 0;
}
```